packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -185,13 +186,29 @@ const extractInnerText = (element: HTMLElement | null): string | null => {
 export function getElementName(
   element: HTMLElement,
   // Whether to try to resolve if element name is a result of Fbt translation
-): string | null {
+): ALElementNameResult | null {
   let nextElement: HTMLElement | null = element;
   while (nextElement && nextElement.nodeType === Node.ELEMENT_NODE) {
+    const label = nextElement.getAttribute('aria-label');
+    if (label != null && label !== '') {
+
+        return {text: label, source: 'aria-label'};
+    }
+    const labelledBy = nextElement.getAttribute('aria-labelledby');
```

Comment:
I noticed in the spec that `aria-labeledby` takes precedence over other methods, so you should check this first before `aria-label` to be more accurate: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby#:~:text=aria%2Dlabelledby%20takes%20precedence%20over%20all%20other%20methods

Furthermore, according to spec, there could be multiple values separated by space that should be picked up and combined. 

There is an alternative as well `aria-descript` and `aria-describedby` which has similar meaning.: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description